src/main/java/com/google/impactdashboard/configuration/Constants.java

```diff
@@ -7,7 +7,8 @@
   public static final String PROJECT_ID = "concord-intern";
 
   /** The path to the service account key. */
-  public static final String PATH_TO_SERVICE_ACCOUNT_KEY = "/usr/local/google/home/carolinelui/Downloads/key.json";
+  public static final String PATH_TO_SERVICE_ACCOUNT_KEY = 
```

Comment:
is this necessary anymore?

Reply:
yes, we still want to be able to test things locally.